core/include/vecmem/utils/impl/tuple.ipp

```diff
@@ -55,7 +55,7 @@ struct tuple_get_impl<0> {
 }  // namespace details
 
 template <std::size_t I, typename... Ts>
-VECMEM_HOST_AND_DEVICE inline constexpr const auto &get(
+VECMEM_HOST_AND_DEVICE constexpr const auto &get(
```

Comment:
Any particular reason to get rid of these inlines?

Reply:
SonarCloud seems very adamant about this issue.

https://sonarcloud.io/project/issues?issues=AZMFu6RHKtxsJUM8k-A0&open=AZMFu6RHKtxsJUM8k-A0&id=acts-project_vecmem

I was just trusting it on this one for the best practice.